runtime/moonbase/tests/integration_test.rs

```diff
@@ -2932,37 +2932,37 @@ mod fee_tests {
 
 			assert_eq!(
 				sim(1_000_000_000, Perbill::from_percent(0), 1),
-				U256::from(999_000_500),
+				U256::from(998_002_000),
```

Comment:
Where do these hardcoded values come from? If we use a formula maybe compute expected value with the formula?